spec/requests/password_reset_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "spec_helper"
+
+# Using request spec b/c Authlogic won"t work with controller spec
+describe PasswordResetsController, type: :request do
+  let(:user) { create(:user) }
+
+  context "if not logged in" do
+    it "new password reset page should load" do
+      get new_password_reset_path
+      expect(response).to be_success
+    end
+
+    it "submitting password reset form should reset perishable token and send email" do
```

Comment:
Can we add a context block for 'user with email'? took me a while to figure that out.